js_modules/dagster-ui/packages/ui-core/src/assets/AssetTableFragment.tsx

```diff
@@ -13,6 +13,10 @@ export const ASSET_TABLE_DEFINITION_FRAGMENT = gql`
     hasMaterializePermission
     partitionDefinition {
       description
+      dimensionTypes {
+        type
+        dynamicPartitionsDefinitionName
+      }
```

Comment:
Now loading this for each asset displayed in the asset table -- I looked at the backend graphql resolvers and this seems like a simple property retriever so I don't think the additional fields will slow down the page.